src/formats/MMTF.cpp

```diff
@@ -111,7 +128,45 @@ void MMTFFormat::read(Frame& frame) {
         frame.set("deposition_date", structure_.depositionDate);
     }
 
+    // A function to translate from the index in MMTF lists to an atom id
+    // suitable for chemfiles: starts at 0 for each model, and correspond to
+    // the initial atom index if it exists.
+    std::function<size_t(size_t)> atom_id;
+    if (!mmtf::isDefaultValue(structure_.atomIdList)) {
+        atom_id = [&](size_t global_id){
```

Comment:
Nitpick: I prefer explicitly listing values captured by a lambda.

Reply:
I can not since the lambda is capturing `this`. But I moved it to a separate private member function.

Reply:
True, I was reading fast.